spec/models/timeline_entry_spec.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+RSpec.describe TimelineEntry do
+  describe "#entry_type" do
+    TimelineEntry::ENTRY_TYPES.each do |type|
+      it "`#{type}` has a translation" do
+        I18n.t!("documents.history.entry_types.#{type}")
```

Comment:
I think your meaning of this test might be clearer if you used the `exists?` method: https://github.com/svenfuchs/i18n/blob/master/lib/i18n.rb#L200